subprojects/core-api/src/main/java/org/gradle/api/artifacts/ExternalModuleDependency.java

```diff
@@ -20,18 +20,19 @@
  */
 public interface ExternalModuleDependency extends ExternalDependency {
     /**
-     * Returns whether or not Gradle should always check for a change in the remote repository.
+     * Returns whether Gradle may check for a change in the remote repository
```

Comment:
```suggestion
     * Indicates that the given dependency can have different content for the same identifier.
```

Reply:
I honestly like the word 'change' or 'update' better.  'different content' could be understood as in 'you can use different feature variants of the same library at the same time'..

How about:
'Indicates that the contents of the remote library may be changed independently, without altering its identifier, including its version'